### ISSUE TITLE
Pending Action Persists after Cancelling to Sign A Task Comment

### ISSUE THREAD

**Issue by auryn-macmillan** [JoinColony/colonyDapp#1718]
## Steps to reproduce

1. Log in to Colony with metamask
2. Comment on a task
3. Confirm the message in the wallet popover
4. Reject signing the message in metamask
5. Try confirming the message again in the popover

## Expected behaviour

Confirming the message again should re-prompt metamask to sign the message.

## Actual behaviour

Clicking `confirm` after first having rejected signing the message in metamask has no effect.
![on0Lk2cFSseCLVse8Xxx0w](https://user-images.githubusercontent.com/8453294/63034063-a682dd00-be86-11e9-9111-e0f9637f7bf5.gif)

## Suggestions (optional)

Some way to cancel unconfirmed messages in the wallet popover might be useful.

## Feedback Portal Posts
https://help.colony.io/hc/en-us/community/posts/360033700854-Pending-Action-Persists-after-Cancelling-to-Sign-A-Task-Comment

**Comment by chmanie** [comment]
OK, this was tricky. Turns out, it's quite hard to resume these in the current setup. I settled on them just failing, so that the user can try again (the form field won't be deleted, they can just press "Comment" again and get presented a new message to sign). I know this isn't the best result but I feel like it's good enough for now (as making these resumable will incur a rewrite of most of the logic which seems undesirable at this point)
